helm/ray/values.yaml

```diff
@@ -43,9 +43,9 @@ head:
     #   requests:
     #     cpu: "500m"
     #     memory: "512Mi"
-  init_container_enabled: False
+  init_container_enabled: false
   initContainer:
-    arguments: ["chown", "-R", "777", "/tmp/ray-workflows"]
+    arguments: ["chmod", "-R", "777", "/tmp/ray-workflows"]
```

Comment:
🤔

Reply:
ohh yes I remember that this was causing an issue when building the cli and I think I forgot to fix this here back then.

Reply:
Yeah this also differing between the `values.yaml` file and the `values.yaml.tpl` file. I adjusted the `values.yaml`, but we probably need a different solution there. In case this happens, I generally adjust `securityContext` to allow the pod to run as `user: 0`, which solves the issue of it complaining about permissions as well.